docs/guides/migrate-from-spectral.md

```diff
@@ -0,0 +1,131 @@
+
+The first step is to [install Redocly CLI](../installation.md).
+
+## Update command
+
+Replace `spectral lint openapi.yaml` with the equivalent `redocly` command:
+
+```yaml
+redocly lint openapi.yaml
+```
+
+For more information, check out the [`lint` command documentation](../commands/lint.md).
+
+### Specify ruleset
+
+Instead of `--ruleset`/`-r`, use the `--extends` parameter to indicate which ruleset you are using as a basis.
+
+Read more [about linting and rulesets](../api-standards.md)
+
+### Choose output format
+
+Similar to Spectral, Redocly offers multiple output formats using the `--format` parameter.
+
+### Resolvers
+
+If you use `--resolver` to handle how links and remote URLs are resolved, visit the [configuration documentation](../../configuration.md/#resolve-object) to see how to handle this with Redocly.
+
+## Update configuration
+
+The configuration formats are a little different between the tools.
+
+Redocly uses a configuration file called `redocly.yaml`, the main controls for linting are:
+
+* Specify a [ruleset](../rules.md#rulesets).
+* Add configuration for the [rules](../rules.md) accordingly. They can be set to error, warn, or off.
+* Expand the collection with any [configurable rules](../rules/configurable-rules.md) that fit your standard.
+
+### Example Redocly configuration file
+
+Below is an example of a `redocly.yaml` configuration file, enabling the [minimal ruleset](../rules/minimal.md), disabling the `security-defined` rule, and setting up an example [configurable rule](../rules/configurable-rules.md) to check for the word "test" appearing in an operation summary.
+
+```yaml
+extends:
+  - minimal
+
+rules:
+  security-defined: off
+  rule/naming:
+    subject:
+      type: Operation
+      property: summary
+    assertions:
+      notPattern: /test/
+    message: "Operation summary must not include the word test"
+```
+
+It is also possible to configure additional rules for specific APIs using the [APIs object](../../configuration.md/#apis-object) to set per-API rules (or exceptions!).
+
+
+### Redocly rules and Spectral equivalents
+
+Included here is an attempt to map the simliar-but-not-identical naming of rules between the tools. If you spot anything that needs adding or updating, please [tell us](https://github.com/redocly/redocly-cli/issues/new)?
+
+
+| Spectral rules                         | Redocly rules                                 |
+|----------------------------------------|-----------------------------------------------|
+| `duplicated-entry-in-enum`             |                                               |
+| `info-contact`                         | `info-contact`                                |
+| `info-description`                     |                                               |
```

Comment:
I wonder if it would be helpful to include configurable rule equivalents, like:

```yaml
rules:
  rule/info-description:
    subject:
      type: Info
      property: description
    assertions:
      defined: true
```

no-eval:

```yaml
rules:
  rule/no-eval:
    subject:
      type: any
      property: description
    assertions:
      notPattern: 'eval('
```
no-script
```yaml
rules:
  rule/no-script-tag:
    subject:
      type: any
      property: description
    assertions:
      notPattern: '<script'
```

openapi-tags:

```yaml
rules:
  rule/openapi-tags:
    subject:
      type: Tag # or maybe TagList ?
    assertions:
      defined: true
```

operation-tags

```yaml
rules:
  rule/operation-tags:
    subject:
      type: Operation
      property: tags
    assertions:
      defined: true
      maxLength: 1 # I guess we already have `operation-singular-tag`
```

It looks like there are 7 of them in total that don't exist in Redocly rules? Or perhaps open an issue and link to it where people can 👍 if they want a built-in equivalent rule where a simple configurable rule isn't possible? I posted a few up above (untested).

Reply:
I love this and will move it out into another issue so we can publish this basic post soon, and work on expanding it!